README.md

```diff
@@ -3,33 +3,79 @@
+All that plus great docs and sane APIs. Also, this project has been built to be a good citizen in the Grafana k6 ecosystem by:
 
-xk6-disruptor is intended for systems running in kubernetes. Other platforms are not supported at this time.
+- Working well with other extensions (like [xk6-browser](https://github.com/grafana/xk6-browser))
+- Working well with k6's core concepts and features.
 
-The extension offers an [API](./docs/02-api/01-api.md) for creating disruptors that target one specific type of component (for example, Pods) and are capable of injecting different types of faults such as errors in HTTP requests served by that component. Currently disruptors exist for [Pods](/docs/02-api/02-pod-disruptor.md) and [Services](/docs/02-api/03-service-disruptor.md), but others will be introduced in the future as well as additional types of faults for the existing disruptors.
+Don't believe us? Check out the following example!
 
+```js
+import { PodDisruptor } from "k6/x/disruptor";
 
-> ⚠️  xk6-disruptor is in the alpha stage, undergoing active development. We do not guarantee API compatibility between releases - your k6 scripts may need to be updated on each release until this extension reaches v1.0 release.
+export default function () {
+    // Create a new pod disruptor with a selector 
+    // that matches pods from the "default" namespace with the label "app=my-app".
+    const disruptor = new PodDisruptor({
+        namespace: "default",
+        select: { labels: { app: "my-app" } },
+    });
 
-## Use case for xk6-disruptor
+    // Check that there is at least one target.
+    const targets = disruptor.targets();
+    if (targets.length != 1) {
+        throw new Error("expected list to have one target");
+    }
 
-The main use case for xk6-disruptor is to test the resiliency of an application of diverse types disruptions by reproducing their effects, but without having to reproduce their root-causes. For example, inject delays in the HTTP requests an application makes to a service  without having to stress or interfere with the infrastructure (network, nodes) on which the service runs, or affecting other workloads in unexpected ways.
+    // Disrupt the targets by injecting HTTP faults into them for 30 seconds.
+    disruptor.injectHTTPFaults(
+        {
+            averageDelay: 500,
+            errorRate: 0.1,
+            errorCode: 500,
+        },
+        30
+    );
+}
+```
 
-In this way, xk6-disruptor make reliability tests repeatable, predictable and limits their blast-radius. These are important characteristic in order to incorporate this kind of tests in the test suits of applications deployed on shared infrastructures such as staging environments.
+## Features
+
+The project at this time is highly focused on providing an excellent experience to test systems running in Kubernetes. Other platforms are not supported at this time.
```

Comment:
It sounds like a piece of marketing to me. I personally don't feel comfortable using this language in the repository of an open source project.

Reply:
For me the problematic phrase is "excellent experience".

Reply:
Yup. I removed it :+1: